flict/flictlib/arbiter.py

```diff
@@ -41,7 +45,23 @@ def verify_package(self, package, licenses):
         inbound_license = package['license']
         logging.debug(f"   * Inbound license:  {inbound_license}")
 
-        return [self.inbounds_outbound_check(outbound_license, [inbound_license]) for outbound_license in licenses]
+        checks = []
+        problems = []
+        for outbound_license in licenses:
+            check = self.inbounds_outbound_check(outbound_license, [inbound_license])
+            if 'problems' in check and check['problems']:
+                problems += check['problems']
```

Comment:
That could be simplified to `problems += check.get('problems', [])` - which would help readability